quartus/workflow.py

```diff
@@ -85,7 +84,7 @@ def create_tcl_system_file(target, custom_components, sys_clock_rate_hz, templat
             f"package require -exact qsys {quartus_version}\n")
         out_file.write(f"load_system {{{target.base_qsys_file}}}\n")
         out_file.write(
-            f"set_instance_parameter_value pll_using_AD1939_MCLK {{gui_output_clock_frequency0}} {{{sys_clock_rate_hz/1_000_000}}}")
+            f"set_instance_parameter_value pll_using_AD1939_MCLK {{gui_output_clock_frequency0}} {{{sys_clock_rate_hz/1_000_000}}}\n\n")
```

Comment:
So we're setting the PLL output clock frequency automatically based upon the model's system clock frequency? If so, I presume the 1000000 division is to the frequency into the proper units?

Reply:
Yes

Reply:
Cool. Having a comment about the expected units would be good.